game_eggs/minecraft/java/nanolimbo/README.MD

```diff
@@ -0,0 +1,12 @@
+# NanoLimbo
+
+This is lightweight minecraft limbo server, written on Java with Netty. The main goal of the project is maximum simplicity with a minimum number of sent and processed packets. This limbo is empty, there are no ability to set schematic building since this is not necessary. You can send useful information in chat or BossBar.
```

Comment:
Please correct this grammar

("This is a lightweight" or preferably "A lightweight")